k9mail-library/src/main/java/com/fsck/k9/mail/Flag.java

```diff
@@ -1,53 +1,84 @@
-
 package com.fsck.k9.mail;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+
+import timber.log.Timber;
+
+
 /**
  * Flags that can be applied to Messages.
+ *
+ * Terminology:
```

Comment:
If you could make a PR for the website we can merge that in at the same time we do a release :)